bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -1641,19 +1656,29 @@ void reset(final CompletionKey key,
             this.ledgerId = ledgerId;
             this.entryId = entryId;
             this.startTime = MathUtils.nowInNano();
-            this.timeout = scheduleTimeout(key, addEntryTimeout);
         }
 
         @Override
         public void writeComplete(int rc, long ledgerId, long entryId,
                                   BookieSocketAddress addr,
                                   Object ctx) {
-            cancelTimeoutAndLogOp(rc);
+            logOpResult(rc);
             originalCallback.writeComplete(rc, ledgerId, entryId, addr, ctx);
             key.release();
             handle.recycle(this);
         }
 
+        @Override
+        boolean maybeTimeout() {
+            LOG.info("maybeTimeout");
```

Comment:
remove this logging

Reply:
Not related to this review, Since we have already import lombok, how about refine all the LOG by lombok.extern.slf4j.Slf4j to make them more unify and cleaner?

Reply:
@zhaijack Yes, but lets hold off until all the yahoo changes are in. Changes like that can cause a lot of churn and make merges a pain.

Reply:
logging removed